alembic/README.md

```diff
@@ -8,16 +8,17 @@ normally do.
 commands:
 ```
 sep drop_uber_db
-sep alembic upgrade head
+sep alembic upgrade heads
+# If you have only the uber plugin installed, use `alembic upgrade head` instead of `heads`
```

Comment:
OK, I was thinking I'd double-check this before making that change but... the documentation isn't super specific so I'll trust you on this one. :)